src/alto-ui/Dialog/__tests__/Dialog.test.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Dialog from '../Dialog';
+
+import CloseButton from '../../CloseButton';
+
+jest.mock('../../CloseButton', () => {
+  return props => {
+    return <button className="CloseButton" onClick={() => props.onClick()} />;
+  };
+});
+
+describe('Dialog', () => {
+
+  const defaultProps = {
+    id: 'string',
+    // className: 'string',
```

Comment:
If you don't need those than remove all comments

Reply:
Removed